packages/freebsd_port/imunes/pkg-plist

```diff
@@ -35,9 +35,27 @@ lib/imunes/gui/initgui.tcl
 lib/imunes/gui/linkcfgGUI.tcl
 lib/imunes/gui/mouse.tcl
 lib/imunes/gui/nodecfgGUI.tcl
+lib/imunes/gui/selectTheme.txt
+lib/imunes/gui/setidioma.txt
 lib/imunes/gui/theme.tcl
 lib/imunes/gui/topogen.tcl
 lib/imunes/gui/widgets.tcl
+lib/imunes/gui/ayuda/Practica1.pdf
```

Comment:
Your PDF files are not included in any of your commits, are they really needed in default IMUNES? Also, remove them in the Help menu. I think these files should be in your own fork, not here.